pkgs/top-level/all-packages.nix

```diff
@@ -34751,4 +34751,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  lookatme = callPackage ../tools/misc/lookatme {};
```

Comment:
I didn't spot this before, all-packages is sorted alphabetically and not which package is the newest

Reply:
done